redisson/src/main/java/org/redisson/spring/session/RedissonSessionRepository.java

```diff
@@ -198,38 +201,40 @@ public boolean isExpired() {
         public String changeSessionId() {
             String oldId = delegate.getId();
             String id = delegate.changeSessionId();
-            if (redisson.getConfig().isClusterConfig()) {
-                Map<String, Object> oldState = map.readAllMap();
-                map.delete();
-                
-                map = redisson.getMap(keyPrefix + id, map.getCodec());
-                map.putAll(oldState);
-                
-                RBucket<String> bucket = redisson.getBucket(getExpiredKey(oldId));
-                long remainTTL = bucket.remainTimeToLive();
-                bucket.delete();
-                redisson.getBucket(getExpiredKey(id)).set("", remainTTL, TimeUnit.MILLISECONDS);
-            } else {
-                map.rename(keyPrefix + id);
-                redisson.getBucket(getExpiredKey(oldId)).rename(getExpiredKey(id));
-            }
+
+            Map<String, Object> oldState = map.readAllMap();
```

Comment:
These two commands also could be included in the same batch.

Reply:
`map` is already being set and modified in the constructor. I am not sure how to integrate this in the first batch. Now using `batch.getMap(map.toString()).readAllMapAsync();`. Hope this is correct. please let me know. 

Thank you for your advice and time!